src/cephadm/cephadm

```diff
@@ -2749,6 +2749,12 @@ def command_shell():
         mounts[pathify(args.config)] = '/etc/ceph/ceph.conf:z'
     if args.keyring:
         mounts[pathify(args.keyring)] = '/etc/ceph/ceph.keyring:z'
+    if args.mount:
+        mount = pathify(args.mount)
+        filename = ''
+        if os.path.isfile(mount):
+            _, filename = os.path.split(mount)
+        mounts[mount] = '/mnt/{}:z'.format(filename)
```

Comment:
```suggestion
        mounts[mount] = '/mnt/{}:z'.format(os.path.basename(mount))
```

this doesn't work with symlinks, char/block devs, etc.

maybe just use the basename instead?